### PR TITLE
Change installation and packaging to standard python method

### PR DESCRIPTION
**Pull request by bunnylab**
Listed installation method appears to be broken or at least did not work for me. Additionally brew is mac specific, a better alternative is just to use standard python tools like pip for the installation which is compatible with mac and all other platforms. Switches script to being run from a python package with a __main__ so it may be run directly or with the -m switch. 

Also included fixes for a few other nitpicks, for instance needs a license file, no .gitignore, no install_requires in setup.py and etc.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Byte-compiled / optimized / DLL files
```
Comment (AlexEdgcomb) [inline .gitignore:1]: There seem to be excessive ignores here. For example, this repo doesn't have Django or Flask, so don't need the associated gitignores.

**Review by AlexEdgcomb — COMMENTED** [review]
@grahamwthompson , I wonder if the blank `zybook/__init__.py` file might be omitted.

File: setup.py [hunk 2]
```diff
@@ -8,5 +8,7 @@
       author='Max Harley',
       author_email='maxh@maxh.io',
       url='http://github.com/t94j0/zybook',
-      scripts=['zybook']
+      license='wtfpl',
```
Comment (AlexEdgcomb) [inline setup.py]: Probably want to update the license here, if going with the MIT

**Comment by t94j0** [comment]
@grahamwthompson I'm not a huge python user, so I may be doing nothing wrong, but, that doesn't seem to be working for me.

When running `python3 setup.py install`, I get:

```
/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/distutils/dist.py:274: UserWarning: Unknown distribution option: 'install_requires'
  warnings.warn(msg)
running install
running build
running build_scripts
error: [Errno 21] Is a directory: 'zybook'
```

**Comment by bunnylab** [comment]
`pip install -r requirements.txt`  and `python3 zybook`  Should do it.  setup.py install isn't necessary here since you aren't building any packages locally. Nevertheless shouldn't be throwing errors. I'll check that.

Didn't change the setup.py parameter from scripts to packages after updating to package structure and that caused the error when running setup.py. Fixed.

**Comment by t94j0** [comment]
Gotcha. I only have distutils set up so that Homebrew works in one line 😄 . I'll edit the README.md to account for your changes and merge.

**Pull request merged**